Chat App/ViewModel/Pub/PubChatLoginViewModel.swift

```diff
@@ -70,37 +73,37 @@ class PubChatLoginViewModel {
      */
     func loginUser(username: String) {
         shouldShowLoading.accept(true)
-        UserManager.userSignin(username: username, hash: encodedPassword!)
+        userManager.userSignin(username: username, hash: encodedPassword!)
```

Comment:
careful with forcing optionals

Reply:
This issue has been addressed in the most recent commits.